www/views/downloadApp.html

```diff
@@ -0,0 +1,23 @@
+<ion-view id="onboard-welcome" class="onboarding">
+  <ion-content ng-controller="downloadAppController" scroll="false">
+    <div class="onboarding-topic" translate>Download NavPay app</div>
+    <div class="onboarding-illustration-app"></div>
+    <p ng-show="fromSettings">Note: You will need to re-enter your Passphrase to access your wallet on the app version.</p>
```

Comment:
passphrase === wallet seed phrase? If so I think we should be more specific here

Reply:
Yep. Good idea!